hasjob/static/js/app.js

```diff
@@ -369,59 +365,14 @@ window.Hasjob.Filters = {
       isFilterDropdownClosed = true;
     });
 
-    filters.ButtonRactive = new Ractive({
-      el: 'hg-site-nav-toggle',
-      template: '#filters-button-ractive',
-      data: {
-        sidebarOn: false
-      },
-      showSidebar: function() {
-        this.set('sidebarOn', true);
-        filters.ractive.showSidebar();
-      },
-      hideSidebar: function() {
-        this.set('sidebarOn', false);
-        filters.ractive.hideSidebar();
-      },
-      oncomplete: function() {
-        //Search icon on mobile to open/close filters menu
-        $('#hg-site-nav-toggle').click(function(event) {
-          event.preventDefault();
-          if (filters.ButtonRactive.get('sidebarOn')) {
-            filters.ButtonRactive.hideSidebar();
-          }
-          else {
-            filters.ButtonRactive.showSidebar();
-          }
-        });
-
-        // Done button for filters on mobile
-        $('#js-mobile-filter-done').click(function(event) {
-          event.preventDefault();
-          filters.ButtonRactive.hideSidebar();
-        });
-
-        //On pressing ESC, close the filters menu
-        $(document).keydown(function(event) {
-          if (event.keyCode === 27) {
-            event.preventDefault();
-            filters.ButtonRactive.hideSidebar();
-          }
-        });
-
-        window.Hasjob.Body.ractive.on('swipeRight', function() {
-          if (isSlidingMenu && !filters.ButtonRactive.get('sidebarOn')) {
-            filters.ButtonRactive.showSidebar();
-          }
-        });
-
-        window.Hasjob.Body.ractive.on('swipeLeft', function() {
-          if (isSlidingMenu && filters.ButtonRactive.get('sidebarOn')) {
-            filters.ButtonRactive.hideSidebar();
-          }
-        });
+    //On pressing ESC, close the filters menu
+    $(document).keydown(function(event) {
+      if (event.keyCode === 27) {
+        event.preventDefault();
+        filters.dropdownMenu.closeOnMobile();
```

Comment:
Have we covered tapping outside the filters box?

Reply:
Yes, https://github.com/hasgeek/hasjob/pull/429/files#diff-25f7a8564bafd30a472f661a106c6008R232